instrumentor/controllers/sourceinstrumentation/common.go

```diff
@@ -98,97 +98,91 @@ func syncWorkload(ctx context.Context, k8sClient client.Client, scheme *runtime.
 		return ctrl.Result{}, err
 	}
 
-	podWorkload := k8sconsts.PodWorkload{
-		Name:      obj.GetName(),
-		Namespace: obj.GetNamespace(),
-		Kind:      workload.WorkloadKindFromClientObject(obj),
+	if !enabled {
```

Comment:
reviewer note: this was the `else` part of the original test, far below.

Since it's only returning, I moved it up here and the entire section beflow does not need to indent